mac

```diff
@@ -105,6 +105,9 @@ brew bundle --file=- <<EOF
 tap "homebrew/services"
 tap "caskroom/fonts"
 
+# Accept Xcode licence automatically
```

Comment:
Putting the command here means it's actually executed by homebrew as part of the `brew bundle` command a few lines up. I don't think this will work correctly.

Can you move it up to line 63 or 64 so it gets run before we install hombrew?